proposals/repo-to-tar/README.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Package Repository to Tar"
+authors: [ "Ashish Kumar <ashishndiitr@gmail.com>" ]
+status: "draft"
+approvers: [ ]
+---
+
+# <Package Repository to Tar>
+
+## Problem Statement
+
+Presently imgpkg does not support the creation of tar file directly.
+In order to obtain a tar file, one needs to first push the bundle image to the registry and then leverage the [command](https://carvel.dev/imgpkg/docs/v0.37.x/air-gapped-workflow/#option-2-with-intermediate-tarball) for copying as tar in the air gapped workflow.
```

Comment:
Let's talk about the end result in more detail and elaborate on the problem at hand. The problem statement should make it very clear to a rider what we are trying to achieve.

What is this "tar file"? How will this tar file be used?